test_LJAPI.py

```diff
@@ -0,0 +1,27 @@
+# coding=utf-8
```

Comment:
Again, this entire file is duplicate.  Why is all this code in your code base twice?  Which one counts?